apps/accounts/forms/auth.py

```diff
@@ -251,12 +275,12 @@ def clean(self):
                     pass
                 else:
                     self.inactive_user = True
-                    raise forms.ValidationError("That account is not active yet because you haven't confirmed your email. <a id=\"resend_confirmation\" href=\"javascript:void(0);\">Resend email confirmation &gt;</a>")
+                    raise forms.ValidationError("That account is not active yet because you haven't confirmed your email. <a id=\"resend_confirmation\" href=\"{}\">Resend email confirmation &gt;</a>".format(reverse('account_resend_confirmation')))  # noqa
```

Comment:
```suggestion
                    raise forms.ValidationError(
                        "That account is not active yet because you haven't confirmed your email. <a id=\"resend_confirmation\" href=\"{}\">Resend email confirmation &gt;</a>".format(  # noqa
                            reverse('account_resend_confirmation')
                        )
                    )
```

Reply:
Done